

<launch>
  <node pkg="pointcloud_to_grid" type="pointcloud_to_grid_node" name="ptg" output="screen" >
    <!-- center position xy -->
    <param name="position_x" value="-5"/>
    <param name="position_y" value="0"/>
    <!-- size of a grid cell -->
    <param name="cell_size" value="1"/>
    <param name="length_x" value="50"/>
    <param name="length_y" value="100"/>
    <!--
    <param name="cloud_in_topic" value="/os_cloud_node/points"/>
    -->
    <param name="cloud_in_topic" value="/left_os1/os1_cloud_node/points"/>
    <!--
    <param name="frame_out" value="os1_sensor"/>
    -->
    <param name="mapi_topic_name" value="lidargrid_i"/>
    <param name="maph_topic_name" value="lidargrid_h"/>
  </node>
  <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconf" output="screen" />
</launch>
